backend/src/Data/Institution.cs

```diff
@@ -101,4 +101,9 @@ public void Verify()
     {
         State = InstitutionState.VERIFIED;
     }
+
+    public void ChangeOperatingState()
```

Comment:
This name is misleading. Either call it `SwitchOperatingState` and change the implementation such that the operating state switches between `OPERATING` and `NOT_OPERATING`. Or call it something like `ChangeToNotOperating` or `MarkAsNotOperating`. Or add a parameter with the new operating state and set it to that new state (that would just be a setter then).

Reply:
I have changed it to SwitchOperatingState and now the operating stage can be toggled between operating and not operating.